src/searchForm/baseStyles.tsx

```diff
@@ -0,0 +1,50 @@
+import { space, font, color, radius } from '_utils/branding'
+
+const overlayWidth = '375px'
+
+export const overlayBaseStyle = `
+  box-sizing: border-box;
+  width: ${overlayWidth};
+  border: 1px solid ${color.border};
+  border-radius: ${radius.l};
+  padding: ${space.xl};
+  padding-top: ${space.s};
+  /* Same as TripCard, we might need to have box-shadow specs so that we can move it to branding file */
+  box-shadow: 0 1pt 4pt rgba(0, 0, 0, 0.16), 0 2pt 8pt rgba(0, 0, 0, 0.08);
```

Comment:
Maybe just move this to branding file.

Reply:
I directly used `shadow.searchForm` since we already defined it :)